### PR TITLE
chore(timed): update timed-frontend image tag to v3.0.0-rc3

### PR DESCRIPTION
**Pull request by derrabauke**
# Description

# Issues

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated the changelog with an `artifacthub.io/changes` annotation in `Chart.yaml`, check the [example](docs/development.md#Changelog) in the documentation.
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: charts/timed/Chart.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ dependencies:
     condition: postgresql.enabled
 annotations:
   artifacthub.io/changes: |
-    - kind: changed
-      description: "updated timed-backend image tag"
+   - kind: changed
+      description: "updated timed-frontend image tag"
```
[the remark this answers is not in the document]
Comment (derrabauke) [reply charts/timed/Chart.yaml:26]: Thanks! This is truly confusing :confused:

**Comment by derrabauke** [comment]
@trowik you know what's wrong with [that](https://github.com/adfinis/helm-charts/actions/runs/3995231348/jobs/6853863215#step:12:1212)?
> 2023-01-24 10:29:44.204 GMT [176] ERROR: column "estimated_time" of relation "projects_project" already exists)?

**Comment by hairmare** [comment]
> > 2023-01-24 10:29:44.204 GMT [176] ERROR: column "estimated_time" of relation "projects_project" already exists)?

might be related to this:
* https://github.com/adfinis/timed-backend/issues/878

i triggered a re-run to be sure it isn't racy

**Comment by derrabauke** [comment]
@hairmare [Let's put the `timed-frontend` image on ghcr.io too.](https://github.com/adfinis/timed-frontend/pull/780) Any objections?

**Comment by hairmare** [comment]
> @hairmare [Let's put the `timed-frontend` image on ghcr.io too.](https://github.com/adfinis/timed-frontend/pull/780) Any objections?

not at all, i approved you gh actions PR

**Comment by hairmare** [comment]
@derrabauke looks like it didn't push a tagged image because the tag already existed, easiest way forward is probably to tag another release so gh actions can do it's thing

**Comment by derrabauke** [comment]
> @derrabauke looks like it didn't push a tagged image because the tag already existed, easiest way forward is probably to tag another release so gh actions can do it's thing

Thanks again for helping me with this struggle. ;) Will push another tag.

**Review by hairmare — APPROVED** [review]
lgtm :shipit: 

(well, with one more rebase that can be done via the gui and shouldn't need re-approval)

**Pull request merged**